pom.xml

```diff
@@ -93,6 +93,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    
+    <dependency>
+      <groupId>com.typesafe.play</groupId>
+      <artifactId>play-json_2.11</artifactId>
+      <version>2.6.9</version>
+    </dependency>
+    <dependency>
+      <groupId>com.typesafe.akka</groupId>
+      <artifactId>akka-http_2.11</artifactId>
+      <version>10.0.11</version>
```

Comment:
I'm a bit surprised this version is different from the akka.version string - can you elaborate on why?

Reply:
Looks like Akka-HTTP and (main) Akka have totally different versioning pattern. From Akka docs:
> Akka HTTP is provided as independent modules from Akka itself under its own release cycle. Akka HTTP is compatible with Akka 2.5 and any later 2.x versions released during the lifetime of Akka HTTP 10.1.x.